src/components/change-wallet/AddressRow.tsx

```diff
@@ -121,22 +119,25 @@ interface AddressRowProps {
 }
 
 export default function AddressRow({ contextMenuActions, data, editMode, onPress }: AddressRowProps) {
-  const nativeCurrency = useSelector((state: AppState) => state.settings.nativeCurrency);
   const notificationsEnabled = useExperimentalFlag(NOTIFICATIONS);
 
-  const { address, balance, color: accountColor, ens, image: accountImage, isSelected, isReadOnly, isLedger, label, walletId } = data;
```

Comment:
the source of `balance` here comes from `useWalletsWithBalancesAndNames` which uses `useWalletBalances` which we should either delete/clean up or update to pass along the balance in this new way.